solutions/swb-reference/SWBv2.openapi.json

```diff
@@ -0,0 +1,2541 @@
+                    "example": "Example Project 1"
+                  },
+                  "name": {
+                    "type": "string",
+                    "example": "Example Project 1"
+                  }
+                }
+              },
+              "example": {
+                "costCenterId": "cc-exampleCostCenterId",
+                "description": "Example Project 1",
+                "name": "Example Project 1"
+              }
+            }
+          }
+        },
+        "responses": {
+          "200": {
+            "description": ""
+          }
+        }
+      }
+    },
+    "/projects/{projectId}/environmentTypes/{envTypeId}/configurations/{envTypeConfigId}/relationships": {
+      "put": {
+        "tags": ["projects"],
+        "summary": "Associate project with EnvTypeConfig",
+        "description": "Associate project with EnvTypeConfig",
+        "operationId": "associateProjectWithEnvtypeconfig",
+        "parameters": [
+          {
+            "name": "Cookie",
+            "in": "header",
+            "schema": {
+              "type": "string",
+              "example": "access_token={{ACCESS_TOKEN}};_csrf={{CSRF_COOKIE}}"
+            }
+          },
+          {
+            "name": "csrf-token",
+            "in": "header",
+            "schema": {
+              "type": "string",
+              "example": "{{CSRF_TOKEN}}"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": ""
+          }
+        }
+      },
+      "delete": {
+        "tags": ["projects"],
+        "summary": "Delete Association between project and EnvTypeConfig",
+        "description": "Delete Association between project and EnvTypeConfig",
+        "operationId": "deleteAssociationBetweenProjectAndEnvtypeconfig",
+        "parameters": [
+          {
+            "name": "Cookie",
+            "in": "header",
+            "schema": {
+              "type": "string",
+              "example": "access_token={{ACCESS_TOKEN}};_csrf={{CSRF_COOKIE}}"
+            }
+          },
+          {
+            "name": "csrf-token",
+            "in": "header",
+            "schema": {
+              "type": "string",
+              "example": "{{CSRF_TOKEN}}"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": ""
+          }
+        }
+      },
+      "parameters": [
+        {
+          "name": "projectId",
+          "in": "path",
+          "required": true,
+          "schema": {
+            "type": "string",
+            "example": ""
+          },
+          "description": "ID of project to associate"
+        },
+        {
+          "name": "envTypeId",
+          "in": "path",
+          "required": true,
+          "schema": {
+            "type": "string",
+            "example": ""
+          },
+          "description": "ID of environment type associated with environment Type Configuration"
```

Comment:
nit: same comment as above

Reply:
This one is fine because it is describing the env type id, which is the env type that is associated with the env type config that is getting unassociated.